### DISCUSSION TITLE
Play Queue?

### DISCUSSION THREAD

**Discussion by hkdd** [leinelissen/jellyfin-audio-player#87] [Q&A]
Is there some way to add an album or song to the queue?  It seems like right now the only option is to play an album which replaces the currently queue.  

Maybe I'm just looking at this the wrong way and there's a better way to manage a play queue than I am familiar with in other audio applications, mostly the subsonic ecosystem.

**Comment by leinelissen** [comment]
If you long press on a single song, you'll get a menu in which you can add songs to the end of the queue. This is unfortunately not yet possible for albums, but something I'm willing to consider implementing!

**Comment by hkdd** [comment]
Oh nice, long press works on single songs.  I noticed another Jellyfin media player, Finamp, also doesn't support a play queue either.  But the web based jellyfin media player does support adding albums to a play queue.  

It's really nice to be able to just queue stuff up.  At lot of times I'll think of something as an album is playing but I want to hear the rest of this album before the next starts.  But I don't want to have to wait for it to end then remember what I wanted to play next.  It's also really nice when I'm about to jump in the car, I can get a couple albums queued up ahead of time.  I assume the alternative would be creating playlists in the jellyfin web app just feels like extra effort. 

Anyway, I think it would definitely be a useful feature.

**Reply by leinelissen** [reply]
Yes, creating playlists is the (admittedly poor) way of handling it right now. I still need to look a better of ways of making multiple controls available for both albums and songs. I'll keep this in the back of my head.

**Comment by vincehardy1** [comment]
Thank you for your hard work and I would also love to have the ability to add to queue an entire album but thanks for the tip on adding songs with a long press.
